GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -199,11 +199,11 @@ - (instancetype)init {
     NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
     [notificationCenter addObserver:self
                            selector:@selector(iOSApplicationDidEnterBackground:)
-                               name:NSExtensionHostDidEnterBackgroundNotification
+                               name:@"UIApplicationDidEnterBackgroundNotification"
```

Comment:
You can leave as it was for extension-type watchOS apps, and you can add TARGET_OS_WATCH to the line 161 since these notifications are already registered for.